opm/input/eclipse/Schedule/Well/WDFAC.hpp

```diff
@@ -53,6 +53,8 @@ namespace Opm {
 
         bool useDFactor() const;
         bool useConnectionDFactor() const;
+        bool useWellDFactor() const;
```

Comment:
This function (`useWellDFactor()`) isn't implemented anywhere.  Is that intentional?

Reply:
Nope. Just some leftover code.